### PR TITLE
Add deprecation notice to README.md

### PR DESCRIPTION
**Pull request by kitsane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Panter Dashboard
 
+## Deprecation Notice
+This is the old dashboard.panter.ch and was replaced by
+<git@git.panter.ch:open-source/dashboard.panter.ch.git>
```
Comment (koffeinfrei) [inline README.md:5]: Better link to the web url

**Comment by koffeinfrei** [comment]
Please also add `[DEPRECATED]` or `[MOVED]` to the repo description with the new web url.

Actually this change to the README is not necessary IMO, the description update is enough.

**Comment by kitsane** [comment]
Ok, I close the pr and will add the notice after the new dashboard is live.

The repo description afterwards will look something like:

```
[DEPRECATED] new dashboard is at https://git.panter.ch/open-source/dashboard.panter.ch
```

**Comment by koffeinfrei** [comment]
@kitsane i'm sorry, you're actually right about this. As this is a public repo the README update makes sense.

**Pull request closed without merging**